python/proton/_events.py

```diff
@@ -57,7 +57,6 @@ def more(self) -> bool:
         return pn_collector_more(self._impl)
 
     def pop(self) -> None:
-        ev = self.peek()
```

Comment:
This is syntax error, method without a body

Reply:
ah, never mind, there is one more statement below; so this was removed because `ev` was unused and `self.peek()` only returns value without doing any side-effects